include/packetgraph/print.h

```diff
@@ -34,6 +34,8 @@ enum pg_print_flags {
 	PG_PRINT_FLAG_BRICK = 8,
 	/* Print raw packet data. */
 	PG_PRINT_FLAG_RAW = 16,
+	/* Print in pcap format */
+	PG_PRINT_FLAG_PCAP = 32,
```

Comment:
Maybe add some doc about this flag: if enabled, this flag disable other print flags